### ISSUE TITLE
New taxonomy requires manual cache refresh to show title

### ISSUE THREAD

**Issue by johncarter-** [statamic/cms#1982]
Create a new taxonomy "on the fly" by typing the new name into the input field, via the entry edit screen.

Then in a template use this antlers code to display a list of all categories and links:
```
{{ taxonomy:categories collection="YOURCOLLECTION" as="categories" }}
  {{ if total_results > 0 }}
  <div class="w-full order-3 md:order-1 mb-8">
    <h3 class="h5 text-gray-900 mb-2">Categories</h3>

    <div class="flex flex-wrap">
      <div class="w-full rounded md:bg-gray-100 md:p-4">
        {{ categories }}
        <p class="mb-1"><a class="py-1 leading-none text-brand-800 hover:underline" href="{{ url }}">{{ title }}</a></p>
        {{ /categories }}
      </div>
    </div>
  </div>
  {{ /if }}
  {{ /taxonomy:categories }}
```

This results in the `{{ title }}` showing the term slug. e.g. "new-category"

**After manually clearing the cache**, {{ title }} shows the title e.g. "New Category".

**Statamic version:** 3.0.0.-beta-31

**PHP version:** 7.4

**Forge Provisioned DO droplet**

**Comment by goellner** [comment]
related to #1981 

cleared the cache, now the title is correct

**Comment by rosswintle** [comment]
I think I'm having similar issues with slugs of terms showing in the control panel...

![Screenshot 2020-06-26 at 10 45 49](https://user-images.githubusercontent.com/1532368/85844245-50671180-b79a-11ea-8135-292a80724dd1.png)

Deleted terms are also not deleting from the stache it would seem.

Seems to be fixed by clearing the stache. More in this video demonstrating multiple issues with terms: https://share.getcloudapp.com/BluODGdp

**Comment by mikemartin** [comment]
@rosswintle Hoping taxonomies get some love soon. 👍 
![2020-08-06 18 12 12](https://user-images.githubusercontent.com/414211/89508253-04839f80-d811-11ea-9950-205ee44a2320.gif)

**Comment by jasonvarga** [comment]
The initial part of this issue is fixed in #2684 but I can't reproduce the title not appearing when you create a term on the actual taxonomies area (like @mikemartin is showing). Is that still happening for you?

**Comment by jasonvarga** [comment]
Nevermind, it happened 😄
